docs/user-guide/images.md

```diff
@@ -69,6 +69,20 @@ images in the ECR registry.
 The kubelet will fetch and periodically refresh ECR credentials.  It needs the
 `ecr:GetAuthorizationToken` permission to do this.
 
+Requirements:
+
+- You must be using kubelet version `v1.2.0` or newer.  (e.g. run `/usr/bin/kubelet --version=true`).
```

Comment:
How about:

- You must be using kubelet version `v1.2.0` or newer. Run this to check:

  ```shell
  kubelete --version=true
  ```